.github/workflows/docker-image-scan.yml

```
@@ -0,0 +1,32 @@
+# Dockerfileで定義されているDocker ImageをTrivyでスキャンし、GitHub Security にスキャン結果を送信します。
+
+name: trivy scan
+
+# プッシュ時に実行する
+on: push
```

```suggestion
on:
  push:
    branches:
      - master
  pull_request:
```
`trivy-action` のドキュメントにしたがったものに戻しておいてほしいです

---

Fixed https://github.com/approvers/OreOreBot2/pull/612/commits/4a6effe0b7b35d1742ee52b96902f2c5067b082e